examples/proxy/README.md

```diff
@@ -25,7 +25,7 @@ module "tfe_node" {
   fqdn                 = "<Fully qualified domain name>"
   ssl_certificate_name = "<Name of the SSL certificate provisioned in GCP>"
   dns_zone_name        = "<Name of the DNS zone in which a record set will be created>"
-  proxy_ip             = "<IP address of the existing proxy>"
+  http_proxy_uri_authority             = "<host and port of the existing proxy>"
```

Comment:
This may have messed up the spacing on this block.

Reply:
Fixed with https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/135/commits/397ad97c0a35d955d143ad15d2b01927e2166ea8.